### ISSUE TITLE
Modulo operation with 0 and 1 is allowed

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#791]
# Bug report

## What's wrong

This code now does not raise a `WPS345` violation:
```python
x % 0
x % 1
```

https://rust-lang.github.io/rust-clippy/master/index.html#modulo_one

**Comment by sobolevn** [comment]
Related #775
